hypothesis-python/src/hypothesis/internal/conjecture/data.py

```diff
@@ -47,9 +47,12 @@ def __repr__(self):
 class Example(object):
     depth = attr.ib()
     label = attr.ib()
+    index = attr.ib()
     start = attr.ib()
     end = attr.ib(default=None)
+    trivial = attr.ib(default=True)
```

Comment:
My reading is that `trivial` generally means "minimal example" (eg all zeros), so I would expect the default to be False.  How does this actually work?  Adding an inline comment to help future readers would be appreciated!

Reply:
I'll add a comment to explain, but basically examples start out trivial and then get marked non-trivial when something interesting happens.